vendor/assets/manifest.webapp

```diff
@@ -32,6 +32,18 @@
       "folder": "/",
       "index": "index.html",
       "public": false
+    },
+    "/": {
+      "folder": "/create",
+      "index": "create.html",
```

Comment:
I think that `create` is not explicit enough. `service-create` or something mentionning the _service_ aspect should be more explicit.